lib/init/config-initializer.js

```diff
@@ -320,10 +321,20 @@ function askInstallModules(modules, packageJsonExists) {
             result(input) {
                 return this.skipped ? null : input;
             }
+        },
+        {
+            type: "select",
+            name: "packageManager",
+            message: "Which package manager does your project use?",
+            initial: 0,
```

Comment:
just wondering if it can be a smarter default by checking how it was run. e.g
```
`npm init/npx`  => npm
`yarn create` => yarn
```

Reply:
We can probably do that by checking if we already have an existing package.lock or yarn.lock or pnpm lock file and default to package managers. 
Also, we might want to change the text to `Which package manager do you want to use?`

Reply:
but someone may don't use lockfile, or the lockfile can be configured a different name?

the text change lgtm. 👍

Reply:
Yep. Also I've seen some projects with both package.lock and yarn.lock as well. At that time we might want to default to one of the two package managers 😅 .

Reply:
I don’t think there’s a good way to detect this. Just best to ask.